src/config/syscheck-config.c

```diff
@@ -110,13 +110,9 @@ int initialize_syscheck_configuration(syscheck_config *syscheck) {
     syscheck->max_fd_win_rt                   = 0;
     syscheck->registry_nodiff                 = NULL;
     syscheck->registry_nodiff_regex           = NULL;
-    syscheck->enable_registry_synchronization = 1;
```

Comment:
Finally, we will keep this functionality so please, revert every change related to this setting in this pull request.